### ISSUE TITLE
Release automation workflow failed 

### ISSUE THREAD

**Issue by givvemee** [YOCOING/YOCO#54]
name: "🐛 Bug report"
description: Report a reproducible bug or regression
body:
  - type: markdown
    attributes:
      value: |
        Thanks you for taking the time to raise up new issue for us!

        This issue tracker is for reporting reproducible bugs or regression's found in [YOCO](https://github.com/YOCOING/YOCO/issues)

        The more information you fill in up, we can take more actions to resolve reported bug.
  - type: textarea
    id: description
    attributes:
      label: Description
      description: Please enter an explicit description of your issue
    validations:
      required: true
  - type: textarea
    id: steps
    attributes:
      label: Reproduction
      description: Please provide detailed steps for reproducing the issue.
      placeholder: |
    validations:
      required: true
  - type: textarea
    id: expected
    attributes:
      label: Expected output
      description: A clear and concise description of what you expected to happen.
      placeholder: |
        As a user, I expected ___ behavior but i am seeing ___
    validations:
      required: true
  - type: dropdown
    attributes:
      label: How often does this bug happen?
      description: |
        Following the repro steps above, how easily are you able to reproduce this bug?
      options:
        - Every time
        - Often
        - Sometimes
        - Only once
  - type: textarea
    id: screenshots_or_videos
    attributes:
      label: Screenshots or Videos
      description: |
      placeholder: |
        You can drag your video or image files inside of this editor ↓
  - type: textarea
    id: additional
    attributes:
      label: Additional context
      description: Add any other context about the problem here.
